.github/workflows/ci.yml

```diff
@@ -143,38 +143,3 @@ jobs:
         uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
 
       - run: shellcheck $(find scripts/ -name '*.sh')
-
-  airgap-e2e-test:
-    name: Airgap E2E test
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
-      - name: Run registry
-        run: |
-          export CONTAINER_ID=$(docker run -d -p 5000:5000 --name registry registry:2)
-          echo "CONTAINER_ID=${CONTAINER_ID}" >> $GITHUB_ENV
-      - name: Install kwctl
-        run: cargo install --locked --path .
-      - name: Save policies
-        run: ./scripts/kubewarden-save-policies.sh --policies-list tests/data/airgap/policies.txt --policies policies.tar.gz
-      - name: Remove policies from store
-        run: |
-          kwctl rm registry://ghcr.io/kubewarden/tests/pod-privileged:v0.1.9
-          kwctl rm https://github.com/kubewarden/pod-privileged-policy/releases/download/v0.1.6/policy.wasm
-      - name: Load policies
-        run: |
-          ./scripts/kubewarden-load-policies.sh \
-              --policies policies.tar.gz \
-              --policies-list tests/data/airgap/policies.txt \
-              --registry localhost:5000 \
-              --sources-path tests/data/airgap/insecure.yml
-      - name: Verify policies in local registry
-        run: |
-          kwctl pull registry://localhost:5000/kubewarden/tests/pod-privileged:v0.1.9 \
-              --sources-path tests/data/airgap/insecure.yml
-          kwctl pull registry://localhost:5000/kubewarden/pod-privileged-policy/releases/download/v0.1.6/policy.wasm \
-              --sources-path tests/data/airgap/insecure.yml
-      - name: Clean up - delete registry
-        if: always()
-        run: |
-          docker rm -f ${{ env.CONTAINER_ID }}
```

Comment:
Cannot we just call the new test here? As far as I can remember, we add this test because the scripts failed due some regression just because we forgot to update them. I would like to avoid user to be the ones whom find this issues. Maybe just set the job as optional in the PR checks.

Reply:
@jvanz the test is running as a part of the e2e tests now, no need to add a new job for it